src/components/Home.jsx

```diff
@@ -0,0 +1,22 @@
+function Home() {
+
+    return (
+      <>
+        <section>
+          <p>
+            [Stock Pic 1]
```

Comment:
might make sense to actually add some pictures here :') visually it makes no sense to the user and looks like the image just didn't load